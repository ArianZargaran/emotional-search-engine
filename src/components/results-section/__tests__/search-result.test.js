import React from "react";
import SearchResult from "../search-result";

it('renders SearchResult correctly', () => {
  const children = {
    title: `This is a mosk query`,
    url: `www.LoremIpsum.com`,
    description: "Coordinates: 36Â°17â€²56.4â€³N 30Â°21â€²17.3â€³Eï»¿ / ï»¿36.299000Â°N 30.354806Â°Eï»¿ / 36.299000; 30.354806 Gagae (Greek: Î“Î¬Î³Î±Î¹), was a town on the southeast coast of Lycia, in what is now the province of Antalya, from which the Gagates lapis derived its name. The ruins are located in Kumluca district, Antalya Province, Turkey.",
    emotion: "ðŸ˜’ 10% | ðŸ˜ƒ 50% | ðŸ˜± 35% | ðŸ™ˆ 5% | ðŸ˜¡ 0%"
  };
  const tree = shallow(<SearchResult children={children} />)
  expect(tree).toMatchSnapshot();
});