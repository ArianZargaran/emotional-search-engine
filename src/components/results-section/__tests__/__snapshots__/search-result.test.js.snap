// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchResult correctly 1`] = `
<li
  className="search-result"
>
  <h1
    className="search-result_title"
  >
    No title was found
  </h1>
  <a
    className="search-result_url"
    href="This Site has no URL"
  >
    This Site has no URL
  </a>
  <p
    className="search-result_description"
  >
    No description available
  </p>
</li>
`;
